Create Proposal for New Chapter Policy

Mariatta opened the pull request:
Closes #9

Mariatta committed:
Create Proposal for New Chapter Policy

terezaif commented on policies/new_chapter_policy.md:
```diff
@@ -0,0 +1,136 @@
+
+- share news from global PyLadies leadership to their members
+
+- organize at least one meetup per year (?)
```
- not sure how to phrase it.. I would say that the chapter should aim at having regular meet-ups, no less often than one every two months. The chapter lead is responsible for managing that that happens by making sure that at any given time there are enough active co-organizers.

amcasari replied:
I agree that have "regularly" scheduled meetups should be a requirement, but I am hesitant to subscribe a cadence. Some meetups don't hold events during seasonal times that are busy for members (like holiday periods). I like the X events per year approach, which I feel gives more leeway. Maybe 4/year (averages out to 1 event per 3 months)?

amcasari replied:
We could also state this as "organizes or supports co-organization of at least N meetups per year" which allows community members to organize events to support the minimum N.

Mariatta replied:
I'm going with at least 1 /year for now, as I think it is generally achievable without demanding too much from volunteers.

terezaif commented on policies/new_chapter_policy.md:
```diff
@@ -0,0 +1,136 @@
+- must be local (city-based). e..g Vancouver instead of BC or Canada
+
+- namespace must be descriptive (`kuala-lumpur` instead of `kl`)?
+
```
I would add also that they have joined slack and have contacted another chapter for "starter mentorship" 

.. some of the chapters in Africa reached out to us with questions when they started.. I think it helps if people are explicitly aware that this is encouraged..

Mariatta replied:
the starter mentorship channel sounds great idea!

amcasari replied:
I love this idea. Is "Starter Mentorship" going to be an official pairing or volunteer effort?

Mariatta replied:
At the moment I think official pairing will be difficult to organize. I was thinking to just start a #prospective-organizers channel on slack, it will be open so anyone can join and ask questions? Other experienced organizers can help and answer. What do you think?

terezaif replied:
I would also go for that.. just to make it clear to people that help is available.. especially for the ones that are too shy to ask otherwise

Mariatta replied:
I've created the #prospective-organizers channel and have added an item to say they're advised to first join that channel. Please re-read and let me know if the phrasing can be improved.

terezaif commented on policies/new_chapter_policy.md:
```diff
@@ -0,0 +1,136 @@
+
+- they will have @pyladies.com email address
+
+- they can have pyladies subdomain
```
to add that the chapter will be listed on the pyladies locations

and that they get the meetup pro membership reimbursed..

Mariatta committed:
mention about meetup pro network, and listing in pyladies locations

amcasari commented on policies/new_chapter_policy.md:
```diff
@@ -0,0 +1,136 @@
+5. What questions do we ask in our form?
+6. How do we review new chapter requests?
+7. For accountability, how do we document new chapter creations?
+
```
Is this the place to discuss when chapters go stale or new organizers request to take over chapters that were once established but not active? Or is transfer/sunset of chapters discussed elsewhere (given that this is a "New Chapter" policy)?

Mariatta replied:
I have another task for "inactive chapter" https://github.com/pyladies/project-admin-logisitics/issues/10 

I wanted to start small with this proposal, and focus on the "new" requests. However, if we think it makes sense to include process surrounding chapter transfer, I'm open to that.

amcasari commented on policies/new_chapter_policy.md:
```diff
@@ -0,0 +1,136 @@
+
+## 1. Why is this policy necessary?
+
+- We had many inactive chapters, that never took off since they were created
```
Missing punctuation for the sentence end.

amcasari commented on policies/new_chapter_policy.md:
```diff
@@ -0,0 +1,136 @@
+
+### 4. What kind of support can new chapter leaders expect from Global PyLadies leadership team?
+
+- as a chapter they can use PyLadies logo and name
```
Is this where we want to state they can create their own local PyLadies logo variants within whatever creative guidelines we have (and link to those guidelines)?

Mariatta replied:
sure, I can link to https://github.com/pyladies/pyladies-assets which has the usage policy info.

amcasari commented on policies/new_chapter_policy.md:
```diff
@@ -0,0 +1,136 @@
+
+- access to organizers channel in slack
+
+- access to organizers resources
```
This should either have a link or be more explicit as to what "Organizers resources" mean.

Mariatta replied:
To me this is just the #organizers channel on slack 😬  (because otherwise everything else is already open/public).

amcasari commented on policies/new_chapter_policy.md:
```diff
@@ -0,0 +1,136 @@
+
+- preferably run by women
+
+- must be local (city-based). e..g Vancouver instead of BC or Canada
```
What are the location considerations for having multiple chapters near each other? For example, New York City is a LARGE area to cover. If a second set of organizers wanted to have a chapter that held events or supported a smaller subset of a city, how do we address that request?

Mariatta replied:
Perhaps a second city chapter is allowed, and will be approved on case-per-case basis?

amcasari commented on policies/new_chapter_policy.md:
```diff
@@ -0,0 +1,136 @@
+
+Prospective organizers will submit new chapter requests via a Google Form.
+
+PyLadies admin & logistics team members will review new chapter requests once
```
It would be nice for submitted requests to have a timeline for review + reply ("We examine requests on the first week of every month and reply to requests within 5 days of review." Setting expectations when you apply into the great unknown is always a better experience.

Mariatta replied:
yes, we used to have a bot that autoreplies with these info, so they got immediate response. We will definitely activate it again once new process in place.

So I think I will need to discuss with the team who will review incoming requests, on when/how often can they review chapter requests, and update the text here.

Mariatta replied:
I've added with timeline of how and when we review chapters. Please check if this is ok?

amcasari reviewed (commented):
Generally thumbs up! Some questions + suggestions. Happy to PR with suggestions for changes, if that's more helpful than comments.

Mariatta committed:
Update PPP 1

Mariatta commented:
Hi all, I've updated some text, additionally I've added the following sections:
- reviewing timeline: how chapter requests will be reviewed, and who are the reviewers.
- when do we expect this policy to take place: I proposed March 1st, 2020
- how folks can ask questions about this proposal: by opening an issue in the repo
- deadline for asking questions/reject this proposal: by Feb 25, because we need to keep moving and continuing.

I'm going to merge this. If folks have questions or have suggestions on wordings, please do open an issue or create a PR with suggested change. Thanks.

amadikwajoyn commented:
I think review the chapters asynchronously and then make a recommendation
